Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.12
+FROM alpine:3.15
 
 ARG KUBE_VERSION="v1.16.10"
```

Comment:
This should be 1.21.x

Reply:
Can you also upgrade the aws cli to the latest on line 5

Reply:
done!